class.leaflet-map.php

```diff
@@ -187,8 +208,8 @@ public static function enqueue_and_register()
         $js_url = $settings->get('js_url');
         $css_url = $settings->get('css_url');
 
-        wp_register_style('leaflet_stylesheet', $css_url, Array(), null, false);
-        wp_register_script('leaflet_js', $js_url, Array(), null, true);
+        wp_register_style('leaflet_stylesheet', $css_url, Array(), LEAFLET_MAP__PLUGIN_VERSION, false);
+        wp_register_script('leaflet_js', $js_url, Array(), LEAFLET_MAP__PLUGIN_VERSION, true);
```

Comment:
adds plugin version like all the other registered scripts; #222

Reply:
Actually I don't think this is right. Adding the plugin version rather than the Leaflet JS version means that if the Leaflet JS version is changed then the cache potentially wouldn't know (if the URL was the same). This is why I would opt for `apply_filters('leaflet_js_version', Leaflet_Map::$leaflet_version)` instead because it would also allow this to be filtered when the version is changed programatically (as we do).

Reply:
Furthermore, $in_footer is set to true here but line 218 is set to false. I think that both need to be true or both need to be false which is why the JavaScript is breaking when using "Merge + Minify + Refresh" plugin.

Reply:
> Actually I don't think this is right. Adding the plugin version rather than the Leaflet JS version means that if the Leaflet JS version is changed then the cache potentially wouldn't know (if the URL was the same). This is why I would opt for `apply_filters('leaflet_js_version', Leaflet_Map::$leaflet_version)` instead because it would also allow this to be filtered when the version is changed programatically (as we do).

This makes me think "null" is better here.

Reply:
> Furthermore, $in_footer is set to true here but line 218 is set to false. I think that both need to be true or both need to be false which is why the JavaScript is breaking when using "Merge + Minify + Refresh" plugin.

Hesitant to change this because we've had issues in the past with people running async and deferred scripts.

Reply:
UPDATE: We have changed our position on this and moved both scripts to be loaded in the header instead of the footer. This is because loading them in the footer breaks compatibility with Leaflet.GestureHandling https://elmarquis.github.io/Leaflet.GestureHandling/